managed/CounterStrikeSharp.API/Modules/Menu/CenterHtmlMenu.cs

```diff
@@ -13,114 +13,73 @@
-                return title[..32];
-            }
-
-            return title;
-        }
-
-        private static string ModifyOptionDisplay(string display)
-        {
-            if (display.Length > 26)
-            {
-                Application.Instance.Logger.LogWarning("Display should not be longer than 26 characters for a CenterHtmlMenu item");
-                return display[..26];
-            }
-
-            return display;
         }
     }
 
     public class CenterHtmlMenuInstance : BaseMenuInstance
     {
+        public override int NumPerPage { get; } = 4;
         private readonly BasePlugin _plugin;
-        public override int NumPerPage => 5; // one less than the actual number of items per page to avoid truncated options
+        private string _message;
+        private Core.Listeners.OnTick _onTick;
 
         public CenterHtmlMenuInstance(BasePlugin plugin, CCSPlayerController player, IMenu menu) : base(player, menu)
         {
             _plugin = plugin;
-            RemoveOnTickListener();
-            plugin.RegisterListener<Core.Listeners.OnTick>(Display);
+            _onTick = DisplayCenterHtmlMenu;
+            _plugin.RegisterListener(_onTick);
         }
-        
-        public override void Display()
+
+        private void DisplayCenterHtmlMenu()
         {
-            if (MenuManager.GetActiveMenu(Player) != this)
-            {
-                Reset();
-                return;
-            }
-            
-            var builder = new StringBuilder();
-            builder.Append($"<b><font color='yellow'>{Menu.Title}</font></b>");
-            builder.AppendLine("<br>");
+            Player.PrintToCenterHtml(_message);
+        }
 
+        public override void Display()
+        {
             var keyOffset = 1;
-
-            for (var i = CurrentOffset; i < Math.Min(CurrentOffset + MenuItemsPerPage, Menu.MenuOptions.Count); i++)
+            var builder = new StringBuilder();
+            builder.Append($"{Menu.Title}<br>");
+            for (var i = CurrentOffset;
+                 i < Math.Min(CurrentOffset + MenuItemsPerPage, Menu.MenuOptions.Count);
+                 i++)
             {
                 var option = Menu.MenuOptions[i];
-                string color = option.Disabled ? "grey" : "green";
-                builder.Append($"<font color='{color}'>!{keyOffset++}</font> {option.Text}");
-                builder.AppendLine("<br>");
+
+                builder.Append(
+                    $"<font color='{(option.Disabled ? "gray" : "green")}'>[!{keyOffset++}]</font> {option.Text}<br>");
             }
-            
+
             if (HasPrevButton)
             {
-                builder.AppendFormat("<font color='yellow'>!7</font> &#60;- Prev");
-                builder.AppendLine("<br>");
+                builder.Append($"<font color='yellow'>[!7] -></font> Prev<br>");
```

Comment:
Why change the arrow direction like the next option?
Keep  `&#60;-` for "<-" instead of "->"